backend/src/packages/chaiNNer_pytorch/pytorch/processing/upscale_image.py

```diff
@@ -71,19 +87,27 @@ def estimate():
 @processing_group.register(
     schema_id="chainner:pytorch:upscale_image",
     name="Upscale Image",
-    description="Upscales an image using a PyTorch Super-Resolution model. \
-            Select a manual number of tiles if you are having issues with the automatic mode. ",
+    description=(
+        "Upscales an image using a PyTorch Super-Resolution model.             Select a"
+        " manual number of tiles if you are having issues with the automatic mode. "
```

Comment:
What is this formatting?

Reply:
That was my auto-formatter going crazy. fixed.